<launch>
    
    <!-- update your path  <Dataset>/polar_oxford_form/ -->
    <param name="seq_dir" type="string" 
        value="/media/psf/Home/Downloads/data_bag/mulran/KAIST02/polar_oxford_form/" />
    <!-- dataset_type "mulran" "4D_Radar_SLAM_dataset" -->
    <param name="dataset_type" type="string" value="4D_Radar_SLAM_dataset" />
    
    <node pkg="radar_slam" type="radarOdometry" name="radarOdometry" output="screen" />

    <!-- <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find yeti)/rviz/yeti_pgo_rviz.rviz" />
    </group> -->

    <node name="bag_player" pkg="rosbag" type="play" args="-r 1 /media/psf/Home/Downloads/data_bag/4D_radar_dataset/Campus_1/campus1.bag" />


</launch>
